db/factories/momentum.txt

```diff
@@ -382,10 +381,6 @@ trigger_tractorbeam
 trigger_userinput
 trigger_vphysics_motion
 trigger_wind
-vgui_indicator_panel
-vgui_movie_display
-vgui_screen
-vgui_slideshow_display
```

Comment:
begrudgingly, I don't believe now is the time to remove VGUI entities - we don't have good alternatives at the moment.